.github/workflows/post-apiview.yml

```diff
@@ -0,0 +1,22 @@
+name: After APIView
+
+on:
+  check_suite:
+    types: [completed]
+
+permissions:
+  statuses: write
+  pull-requests: write
+  checks: read
+
+jobs:
+  after-apiview:
+    name: After APIView
+    runs-on: ubuntu-24.04
+    if: github.event.check_suite.app.name == 'Azure Pipelines'
+    steps:
```

Comment:
I think you will need to add a checkout step.  For performance, use a sparse checkout with minimal fetch-depth.